apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -149,10 +149,42 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
             [fields.phone]: contact.phone,
         })
         setValue(contact)
+        setSelectedContact(contact)
         const isNew = !contact.id
         onChange && onChange(contact, isNew)
     }
 
+    const initialValueIsPresentedInFetchedContacts = fetchedContacts && initialValue && initialValue.name && initialValue.phone && find(fetchedContacts, initialValue)
+
+    const sameAsInitial = (contact) => (
+        initialValue && initialValue.name === contact.name && initialValue.phone === contact.phone
+    )
+
+    const isContactSelected = useCallback((contact, i) => {
```

Comment:
Please, rename this constant.
It's name is like a getter-function, but it has side-effect logic inside.

Reply:
@AntonAL split `<Option />` selection logic into `useEffect` & cleanup callback